lib/oli/delivery/sections/blueprint.ex

```diff
@@ -159,12 +159,13 @@ defmodule Oli.Delivery.Sections.Blueprint do
             "start_date" => nil,
             "end_date" => nil,
             "title" => title,
-            "requires_payment" => false,
-            "payment_options" => "direct_and_deferred",
-            "pay_by_institution" => false,
-            "registration_open" => false,
-            "grace_period_days" => 1,
-            "amount" => Money.new(:USD, "25.00"),
+            "requires_payment" => attrs["requires_payment"] || false,
+            "payment_options" => attrs["payment_options"] || "direct_and_deferred",
+            "pay_by_institution" => attrs[""] || false,
```

Comment:
Should it be `attrs["pay_by_institution"] || false"? If so, perhaps we need to parse the value if it is not a boolean

Reply:
Good catch! I added a test to cover the complete cycle—export and import.